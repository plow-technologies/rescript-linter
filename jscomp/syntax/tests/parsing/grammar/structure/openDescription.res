open Belt 
open! Belt
@onOpen open Belt

open Belt.Array
open! Belt.Array
@onOpen open! Belt.Array
