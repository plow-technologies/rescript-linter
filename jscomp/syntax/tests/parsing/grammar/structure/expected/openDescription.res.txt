open Belt
open! Belt
open Belt[@@onOpen ]
open Belt.Array
open! Belt.Array
open! Belt.Array[@@onOpen ]